history/5279.md

```diff
@@ -0,0 +1 @@
+* @barnson: Enable the WiX native SDK when the new Visual C++ Build Tools bundle is installed. Fixes wixtoolset/issues#5279.
```

Comment:
I'm with Rob here.  I'd like to keep the `committer: WIXBUG:###` nomenclature.  And PR comment instead of commit comment so we can have a link to the issue from the PR.